tests/functional/conftest.py

```diff
@@ -418,6 +432,14 @@ def pyi_builder(tmpdir, monkeypatch, request, pyi_modgraph):
     # The value is same as the original value.
     monkeypatch.setattr('PyInstaller.config.CONF', {'pathex': []})
 
+    def del_temp_dir():
+        if request.node.rep_setup.passed:
+            if request.node.rep_call.passed:
+                if os.path.exists(tmp):
+                    shutil.rmtree(tmp)
+
+    if is_darwin:
```

Comment:
I was only having space issues on the Mac OS X travis workers and was getting errors on the Appveyor workers when this was enabled so I took the easy route and only enabled it for Mac OS X.
See https://ci.appveyor.com/project/matysek/pyinstaller/build/888/job/umaw5usyu3d3gt4m#L1170 for the error.